packages/tocco-ui/src/StatedValue/StatedValue.js

```diff
@@ -35,27 +35,33 @@ const StatedValue = props => {
     label,
     mandatory,
     immutable,
+    type,
     touched
   } = props
 
   const hasError = touched && props.error && Object.keys(props.error).length > 0
   const signal = detectSignal(dirty, hasError)
+  const look = type === 'display'
+    ? 'display'
+    : immutable
+      ? 'immutableField'
+      : 'mutableField'
 
   return (
     <FocusWithin>
       {({focused, getRef}) => {
-        const secondaryPosition = focused || hasValue
+        const secondaryPosition = focused || hasValue || type === 'display'
         return (
           <StyledStatedValueWrapper
             ref={getRef}
             secondaryPosition={secondaryPosition}>
             <StyledStatedValueBox
-              immutable={immutable}
+              look={look}
               signal={signal}>
               {children}
               <StyledStatedValueLabel
-                htmlFor={immutable ? '' : id}
-                immutable={immutable}
+                htmlFor={look === 'mutableField' ? id : ''}
```

Comment:
{...look === 'mutableField' ? { htmlFor:  id}} : {}}  

Like this you can omit the property completely if not needed.